lib/webauthn/public_key_credential.rb

```diff
@@ -57,6 +57,10 @@ def verify(*args)
       true
     end
 
+    def public_key
+      response.credential.public_key
```

Comment:
Can `credential` potentially be `nil` here in case the response is malformed? It's my understanding by seeing [this code](https://github.com/cedarcode/webauthn-ruby/blob/master/lib/webauthn/authenticator_data/attested_credential_data.rb#L35)

Just commenting this because I don't want this accessor to fail hard on the user with a `No method public_key for nil:NilClass`. What do you think?

Reply:
Good call.
I'll handle that.

Reply:
Also, just realized this doesn't work if `response` is a `WebAuthn::AuthenticatorAssertionResponse`. Need to fix that too.

Reply:
👍 good catch

Reply:
Fixed.